src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/EgressTests.cs

```diff
@@ -24,6 +27,7 @@
 using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
+using DiagnosticPortConnectionMode = Microsoft.Diagnostics.Monitoring.UnitTests.Options.DiagnosticPortConnectionMode;
```

Comment:
Why is this necessary? The other existing tests in this file are already using this type.

Reply:
Since you recommended I add the extension for WebApi, I started getting an error for DiagnosticPortConnectionMode that it was an ambiguous reference between Options and WebApi. We had added this when we had the same issue in LogsTests, so I reimplemented it here. I just pushed changes (without changing that), so if you have something else in mind let me know and I'll push out a new version.

Reply:
> Since you recommended I add the extension for WebApi, I started getting an error for DiagnosticPortConnectionMode that it was an ambiguous reference between Options and WebApi. We had added this when we had the same issue in LogsTests, so I reimplemented it here. I just pushed changes (without changing that), so if you have something else in mind let me know and I'll push out a new version.

I reread this a few times and didn't understand what you were referring to until I realized that "extension" is referring to the `using Microsoft.Diagnostics.Monitoring.WebApi;` statement. This makes more sense now.